app/views/items/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1> Items </h1>
+<ul>
+<% @items.each do |item| %>
+  <li>
+    Name: <%= item.name %>
+    Price: <%= item.price %>
+    Supermarket: <%= item.supermarket.name %>
+    Customers Count that bought the item: <%= item.customers.count %>
```

Comment:
Even though it's a simple calculation, to purely adhere to MVC conventions, we would prefer to see this encapsulated in a model method. BEcause it's a calculation of data records, it should live as close to the db as possible, i.e. in a model method. Then, you can call `item.customer_count` here to invoke that method and get the number displayed

Reply:
I'd consider this 1 MVC infraction. The rubric allows for 1 infraction for a passing score! :)

Reply:
I knew I shouldve placed it in the Model! My gut was telling me to do so but I was running short on time and started to panic.

Reply:
That's okay! That's why we allow an infraction in the rubric. Just notes for next time. 

Also next time: include `.distinct` so that you don't count repeat customers!